jackson/src/test/java/com/baeldung/jackson/deserialization/nested/DeserializeWithNestedPropertiesUnitTest.java

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.jackson.deserialization.nested;

```

Comment:
Chris, it looks like you're using tabs for indentation instead of spaces in all three files, so let's adjust them. If possible, you should import and use the standard formatter for the site in your IDE -- for Eclipse, it's in the eclipse module of the tutorials repo.

Reply:
Also, let's update the Jackson dependency to 2.9.2